### PR TITLE
Add util to combine multiple errors so everyone checkable

### PR DESCRIPTION
**Pull request by yah01**
/kind improvement

### CODE REVIEW

File: internal/util/errutil/errors.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	if len(e.errs) == 0 {
+		return nil
+	}
+	return MultiErrors{
+		errs: e.errs[1:],
+	}
+}
+
+func (e MultiErrors) Error() string {
+	final := e.errs[0]
```
Comment (congqixia) [inline internal/util/errutil/errors.go:19]: May cause panic if one `MultiErrors` is Unwrapped to zero errors.

Comment (yah01) [reply internal/util/errutil/errors.go:19]: It should be, as the error of other errors lib (include std) panics if it's nil

Comment (congqixia) [reply internal/util/errutil/errors.go:19]: But you could construct a not nil but zero length  MultiErrors with Unwrap

Comment (yah01) [reply internal/util/errutil/errors.go:19]: Make this type private done

**Review by jiaoew1991 — APPROVED** [review]
/lgtm

**Pull request merged**